src/subject.rs

```diff
@@ -15,8 +15,8 @@ use uuid::Uuid;
 
 /// Subject allows you to attach additional information about your application's environment
 ///
-/// A Subject can be (attached to either a Tracker) where it will be sent with every Event, and/or
-/// attached to an Event, with the Event-level subject taking priority over Tracker-level
+/// A Subject can be attached to either a [crate::Tracker], where it will be sent with every Event, and/or
+/// attached to, and sent with, an Event, with the Event-level subject taking priority over Tracker-level
```

Comment:
I find this rather wordy but it make sense.

Reply:
Reworded slightly to be less of a block of text